srcpkgs/sof-firmware/template

```diff
@@ -26,7 +26,7 @@ do_install() {
 	for arc in {bdw,byt,cht}; do
 		ln -s sof-${arc}-v${version}.ri ${DESTDIR}/usr/${intel_path}/sof/sof-${arc}.ri
 	done
-	for arc in {apl,cnl,icl}; do
+	for arc in {apl,cnl,icl,jsl,tgl}; do
```

Comment:
I couldn't find a "jsl" binary anywhere in the tarball, so it shouldn't be listed here. However, there is an "ehl" binary in the intel-signed folder that should probably be included here.

Reply:
Huh, strange! I just copied jsl as seen in `go.sh` and didn't get any packaging errors, which is why I didn't verify. I removed jsl and added ehl instead!